Backend/src/dtos/metadata.dto.ts

```diff
@@ -6,31 +6,31 @@ export class MetadataDto {
     description: 'Name of this object',
     examples: ['Foo', 'Bar', 'Steven'],
   })
-  name: string;
+  Background: string;
```

Comment:
I think that is a good practice to use lower case in those variable declaration

Reply:
will update